data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
+Credit Card     | Braintree  | 2.9% + $0.30 | At payday  | https://www.braintreepayments.com/pricing
 Paypal          |            | max(2%, $20) | At payday  |
-Bitcoin         |            | 1% + 15¢     | On request |
 
 # https://gratipay.com/about/faq#how-withdraw-funds
+
+
+#### Deserted
+
+What       | When                | Why
+-----------+---------------------+-------------------------
+Balanced   | 2015-06-11          | Service shutdown
+Bitcoin    | 2015-05-15 f20d7506 | Gratipocalypse
```

Comment:
Balanced and Bitcoin are not parallel.

Balanced is parallel to Coinbase.

Bitcoin is parallel to USD.